core/units.h

```diff
@@ -272,13 +273,13 @@ struct units {
 #define SI_UNITS                                                                                           \
         {                                                                                                  \
 	        .length = METERS, .volume = LITER, .pressure = BAR, .temperature = CELSIUS, .weight = KG,  \
-		.vertical_speed_time = MINUTES, .duration_units = MIXED                                    \
+		.vertical_speed_time = MINUTES, .duration_units = MIXED, .show_units_table = false              \
```

Comment:
Why do you want to set (i.e. possibly change) the show units state when switching between metric and imperial?

Reply:
In addition, for the transition, I would set the default behaviour (when then value is not yet specified) to be the old behaviour (i.e. with units).

Reply:
I hesitated here as well because this is misleading. Putting it in the define doesn't mean it is switched when changing between metric, imperial and personalised in the UI.
Otherwise vertical_speed_time and duration_units must not be there as well.
This in fact means that we init show_units_table with default of false.
At least my tests confirm this.